src/Entity/CommitteeMembership.php

```diff
@@ -230,7 +235,7 @@ public function getCommitteeCandidacy(): ?CommitteeCandidacy
         return $this->committeeCandidacy;
     }
 
-    public function setCommitteeCandidacy(CommitteeCandidacy $committeeCandidacy): void
+    public function setCommitteeCandidacy(?CommitteeCandidacy $committeeCandidacy): void
     {
         $this->committeeCandidacy = $committeeCandidacy;
```

Comment:
no need to the reverse association like here https://github.com/EnMarche/en-marche.fr/pull/4461/files#diff-c2c51603d1b5293687e7848efdc5994dR266 ?

Reply:
no need here, as we are on the owner side of the relation